applications/tari_console_wallet/README.md

```diff
@@ -72,15 +72,16 @@ Monitoring 1 sent transactions to Broadcast stage...
 Done! All transactions monitored to Broadcast stage.
 ```
 
-- **list-utxos**
+- **export-utxos**
 
-List all of the unspent transaction outputs (UTXOs) in the wallet.
+Export all the unspent transaction outputs (UTXOs) in the wallet.
 
-`tari_console_wallet --command "list-utxos"`
+`tari_console_wallet --command "export-utxos"`
+`tari_console_wallet --command "export-utxos csv_file <file name>"`
 
 example output:
 ```
-1. list-utxos
+1. export-utxos
 1. Value: 6000 µT OutputFeatures: Flags = (empty), Maturity = 0
 2. Value: 10000 µT OutputFeatures: Flags = (empty), Maturity = 0
 ...
```

Comment:
This is incomplete, 
Explain how the csv works etc. There is no mention of this in the help, only I  command example. 

The example should also be done properly. 


For example:
```
`tari_console_wallet --command "export-utxos"`
example output:

1. export-utxos
1. Value: 6000 µT OutputFeatures: Flags = (empty), Maturity = 0
2. Value: 10000 µT OutputFeatures: Flags = (empty), Maturity = 0
...

`tari_console_wallet --command "export-utxos csv_file <file name>"`
example output:
....
```

Currently reading the text, I don't know which one to give, to provide me that output.